cmd/terramate/cli/script_tree.go

```diff
@@ -75,12 +75,14 @@ func (node *scriptsTreeNode) format(w io.Writer, prefix string, parentScripts []
 	}
 
 	for _, sc := range node.Scripts {
-		desc := exprString(sc.Description.Expr)
 		fmt.Fprintln(w, blockPrefix+"* "+scriptColor(sc.AccessorName()+": "))
 		if sc.Name != nil {
-			fmt.Fprintln(w, blockPrefix+"  "+scriptColor("  name: "+nameTruncation(exprString(sc.Name.Expr))))
+			fmt.Fprintln(w, blockPrefix+"  "+scriptColor("  Name: "+nameTruncation(exprString(sc.Name.Expr), "script.name")))
+		}
+		if sc.Description != nil {
```

Comment:
Can be discussed later, but I would probably leave the description out here, since it might be longer and just be too much information (and maybe even line breaks?) for the tree view.